src/api/GoalsAPI/index.ts

```diff
@@ -0,0 +1,46 @@
+import { db, GoalItem } from '@models';
+import { getJustDate } from '@src/utils';
+
+export const resetDatabase = () => db.transaction('rw', db.goalItems, async () => {
+  await Promise.all(db.tables.map((table) => table.clear()));
+});
+
+export const addGoal = (goalDetails :{
+  title: string,
+  duration: string,
+  repeat: string,
```

Comment:
A goal is either repeating or not repeating. Why don't we use a boolean here?

Reply:
Oh I see that there is already an enum to be used for repeat. Please use that instead of a boolean